CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## moler 1.10.0
+
+### Improved
+* Classes Connection and ObservableConnection are deprecated now.
+
+### Fixed
+* Iperf2 command fails on parsing when used `with -s -P 1` options (single run server)
+
+## moler 1.9.0
+
+### Added
+* New method of converting string to int for whole dict.
+
+### Improved
+* Keep state in background does not block main flow
+* Timestamp is passed to data_received
```

Comment:
`Timestamp is passed to data_received` --> data_received() of commands & events gets additional parameter: time when data was caught on connection

Reply:
Done.